test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -53,6 +53,9 @@ public static CommandRunnerResult Run(string filename, string workingDirectory =
                     process.StartInfo.Environment["NUGET_SHOW_STACK"] = bool.TrueString;
                     process.StartInfo.Environment["NuGetTestModeEnabled"] = bool.TrueString;
                     process.StartInfo.Environment["UseSharedCompilation"] = bool.FalseString;
+                    process.StartInfo.Environment["DOTNET_SKIP_FIRST_TIME_EXPERIENCE"] = bool.TrueString;
+                    process.StartInfo.Environment["DOTNET_CLI_TELEMETRY_OPTOUT"] = bool.TrueString;
+                    process.StartInfo.Environment["SuppressNETCoreSdkPreviewMessage"] = bool.FalseString;
```

Comment:
is this a mistake?  Why wouldn't we want to suppress the .NET (Core) preview message?

Reply:
🤦 Yes, thank you for catching that!